wiki/osu!stream/Song_List/de.md

```diff
@@ -0,0 +1,286 @@
+
+Dieses Songpaket enthält vier Songs, die von dem Internet-Musikkollektiv Odyssey Eurobeat zur Verfügung gestellt wurden.
+
+| Song | Mapper | Schwierigkeitsgrad |
+| :-- | :-- | :-- |
+| Odyssey - Wings of Burning Love | ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328) | ![7/10](img/dif7.png "7/10") |
+| Odyssey - Cherry Blossoms (2010 Mix) | ![][flag_GB] [Mashley](https://osu.ppy.sh/users/41481) | ![5/10](img/dif5.png "5/10") |
+| Mortimer - Rive | ![][flag_US] [ouranhshc](https://osu.ppy.sh/users/139493) | ![5/10](img/dif5.png "5/10") |
+| Travis Stebbins - Magnetic Love | ![][flag_CA] [mm201](https://osu.ppy.sh/users/30655) | ![3/10](img/dif3.png "3/10") |
+
+Siehe auch:
+
+- [Bandcamp-Seite von Odyssey Eurobeat](https://odysseymusic.bandcamp.com/)
+
+## VVVVVV Pack Vol. 1
+
+Preis: **Kostenlos** / **USD$0.99** (vor dem [2020-Update](https://news.osustream.com/post/611013728007274496/osustream-2020-release))
+
+Dieses Songpaket enthält vier Songs von SoulEye, die alle in dem Hit-Plattformspiel [VVVVVV](https://thelettervsixtim.es/) vorkommen.
+
+| Song | Mapper | Schwierigkeitsgrad |
+| :-- | :-- | :-- |
+| Souleye - Positive Force | ![][flag_GB] [RandomJibberish](https://osu.ppy.sh/users/157879) | ![5/10](img/dif5.png "5/10") |
+| Souleye - Pressure Cooker | ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | ![6/10](img/dif6.png "6/10") |
+| Souleye - Potential for Anything | ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328) | ![7/10](img/dif7.png "7/10") |
+| Souleye - Passion for Exploring | ![][flag_CA] [mm201](https://osu.ppy.sh/users/30655) | ![3/10](img/dif3.png "3/10") |
+
+Siehe auch:
+
+- [Webseite von Souleye](http://magnuspalsson.com/)
+
+## SHIKI Pack Vol. 1
+
+Preis: **Kostenlos** / **USD$1.99** (vor dem [2020-Update](https://news.osustream.com/post/611013728007274496/osustream-2020-release))
+
+Dieses Songpaket enthält vier Songs des bekannten BMS-Musikproduzenten SHIKI.
+
+| Song | Mapper | Schwierigkeitsgrad |
+| :-- | :-- | :-- |
+| SHIKI - Lapis | Team STREAM (![][flag_GB] [RandomJibberish](https://osu.ppy.sh/users/157879) und ![][flag_FR] [Kurai](https://osu.ppy.sh/users/98661)) | ![5/10](img/dif5.png "5/10") |
+| SHIKI - Pure Ruby | ![][flag_US] [Lybydose](https://osu.ppy.sh/users/64501) | ![9/10](img/dif9.png "9/10") |
+| SHIKI - BABYLON | ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | ![8/10](img/dif8.png "8/10") |
+| SHIKI - Jade Star | ![][flag_KR] [Nakagawa-Kanon](https://osu.ppy.sh/users/87065) | ![7/10](img/dif7.png "7/10") |
+
+Siehe auch:
+
+- [Webseite von SHIKI](http://shiki2.sakura.ne.jp/)
+
+## Unreleased Tracks Vol. 1
+
+Preis: **Kostenlos** / **USD$1.99** (vor dem [Update v1.1](https://news.osustream.com/post/10624373514/osustream-v11-release))
+
+Dieses Songpaket enthält vier Songs/Arrangements von originalen Touhou-Project-Kompositionen von verschiedenen Musikern aus aller Welt.
```

Comment:
```suggestion
Dieses Songpaket enthält vier Songs/Arrangements von originalen Kompositionen des Touhou Projects von verschiedenen Musikern aus aller Welt.
```
Wouldn't this work instead? `Touhou Project` probably shouldn't have the hyphen anyway since its a proper name :thinking: